### DISCUSSION TITLE
Simple question related to RTA and high-order IFCs

### DISCUSSION THREAD

**Discussion by AUSR-0221** [ttadano/alamode#209] [Q&A]
Kind regards to all,

I want to be sure what kind of order RTA will be used after OPTIMIZE (results with AIMD and so on..), I guess it is up to the third order as I have read from a tutorial related to SCPH shared in other posts here on github..... In RTA I used:

{
.
.
  MODE   = RTA
  FCSXML = AIMD_obtained_by_OPTIMIZATION.xml
  FC2XML = harmonic.xml
.
.
}

And in OPTIMIZE I used:

{
.
.
&interaction
  NORDER = 5  # 1: harmonic, 2: cubic,...
  NBODY = 2 4 4 2 2 
/

&cutoff 
  *-* None 14 12 10 08
.
.
}

**Comment by ttadano** [comment]
In RTA, we usually need harmonic phonon + three-phonon scattering. For this, force constants up to 3rd order are required.
